src/platform_impl/windows/event_loop.rs

```diff
@@ -1664,11 +1664,7 @@ unsafe fn public_window_callback_inner<T: 'static>(
         winuser::WM_SETCURSOR => {
             let set_cursor_to = {
                 let window_state = subclass_input.window_state.lock();
-                if window_state
-                    .mouse
-                    .cursor_flags()
-                    .contains(CursorFlags::IN_WINDOW)
-                {
+                if (lparam & 0xFFFF) == winuser::HTCLIENT {
```

Comment:
`(lparam & 0xFFFF)` -> `LOWORD(lparam)` 
Also, wrapping the condition into a separate let statement and/or short command regarding hit-testing here would be great

Reply:
> command

I assume you meant to write "comment" here.